javaslang-gwt/README.md

```diff
@@ -1,3 +1,10 @@
 # GWT support for Javaslang
 
-Highly Experimental, works and tested in IntelliJ Idea only
+Tested with 535f2fa973995f18d190a689127acbe134a42a15
+
+```
+git checkout 535f2fa9
+ant clean elemental dist-dev
+mvn install:install-file -Dfile=build/lib/gwt-user.jar -DgroupId=com.google.gwt -DartifactId=gwt-user -Dversion=2.8.0-535f2fa9 -Dpackaging=jar 
```

Comment:
Please use the `maven/push-gwt.sh` script instead; it will "unbundle" most dependencies and declare them as Maven dependencies.